src/sugar3/activity/activity.py

```diff
@@ -1212,7 +1317,7 @@ def __get_filters_cb(self):
         }
         filter_dict = dbus.Dictionary(filters, signature='sv')
         logging.debug('__get_filters_cb %r' % dbus.Array([filter_dict],
-                      signature='a{sv}'))
+                                                         signature='a{sv}'))
```

Comment:
If you have cosmetic changes make them on another PR. It makes it hard to see what you changed.